src/shared/mesh/MeshBuilder.hpp

```diff
@@ -73,6 +73,13 @@ class MeshBuilder {
    */
   MeshBuilder( mfem::Mesh&& mesh );
 
+  /**
+   * @brief Grows or shrinks the mesh relative to the origin by the scale factors in the given vector.
+   * @param scale_factors A list of scale factors to apply to each dimension.
+   * @return An rvalue reference to the updated MeshBuilder object.
+   */
+  MeshBuilder&& scale( std::initializer_list<double> scale_factors );
```

Comment:
Minor: I'll keep an eye out to see how/where this is used, but I would have expected a scale or translate function to be a void function, operating on an already constructed MeshBuilder object. Might need to clarify.

Reply:
This lets us chain the mesh operations together